profiles/Shivani1917.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://avatars.githubusercontent.com/u/79411680?s=400&u=44fbd7b8021b67164aed1f19b30db72071400c7c&v=4",
+    "issueId": 1500,
+    "name": "Shivani Badola",
+    "username": "Shivani1917"
```

It should be a two-space indentation and not four.
```suggestion
  "image": "https://avatars.githubusercontent.com/u/79411680?s=400&u=44fbd7b8021b67164aed1f19b30db72071400c7c&v=4",
  "issueId": 1500,
  "name": "Shivani Badola",
  "username": "Shivani1917"
```